samples/tools/autogenbench/scenarios/WebArena/Templates/Orchestrator/scenario.py

```diff
@@ -0,0 +1,240 @@
+import os
+import json
+import testbed_utils
+import autogen
+import evaluation_harness
+import re
+from autogen.agentchat.contrib.orchestrator import Orchestrator
+from autogen.agentchat.contrib.multimodal_web_surfer import MultimodalWebSurferAgent
+from autogen.runtime_logging import logging_enabled, log_event
+
+from evaluation_harness.env_config import ACCOUNTS, GITLAB, MAP, REDDIT, SHOPPING, SHOPPING_ADMIN, WIKIPEDIA, HOMEPAGE
+
+testbed_utils.init()
+##############################
+
+REPLACEMENTS = {
+    "__REDDIT__": REDDIT,
+    "__SHOPPING__": SHOPPING,
+    "__SHOPPING_ADMIN__": SHOPPING_ADMIN,
+    "__GITLAB__": GITLAB,
+    "__WIKIPEDIA__": WIKIPEDIA,
+    "__MAP__": MAP,
+    "__HOMEPAGE__": HOMEPAGE,
+}
+
+# Expand the prompt and the full task
+task_prompt = ""
+TASK = None
+with open("task_prompt.json.txt", "rt") as fh:
+    task_prompt = fh.read()
+with open("task_prompt.json", "wt") as fh:
+    for k in REPLACEMENTS:
+        task_prompt = task_prompt.replace(k, REPLACEMENTS[k])
+    fh.write(task_prompt)
+    TASK = json.loads(task_prompt)
+
+full_task = ""
+with open("full_task.json.txt", "rt") as fh:
+    full_task = fh.read()
+with open("full_task.json", "wt") as fh:
+    for k in REPLACEMENTS:
+        full_task = full_task.replace(k, REPLACEMENTS[k])
+    fh.write(full_task)
+
+# Load the LLM config list
+config_list = autogen.config_list_from_json("OAI_CONFIG_LIST")
+llm_config = testbed_utils.default_llm_config(config_list, timeout=300)
+
+if logging_enabled():
+    log_event(os.path.basename(__file__), name="loaded_config_lists")
+
+assistant = autogen.AssistantAgent(
```

Comment:
question: what is the assistant agent used for?

Reply:
Good question. It can write code and generally solve problems -- eg synthesize info using LLM. We'd need to conduct ablations to see if its actually useful tho.